utils/message_formats/message_formats.py

```diff
@@ -0,0 +1,274 @@
+#!/usr/bin/python3
+#
+# Copyright 2022 SuperDARN Canada
+# Author: Remington Rohel
+from dataclasses import dataclass, field
+import numpy as np
+
+
+@dataclass
+class DebugDataStage:
+    """
+    Defines a stage of debug data (filtered data or antennas_iq data plus associated metadata).
+    """
+    stage_name: str = None
+    main_shm: str = None
+    intf_shm: str = None
+    num_samps: int = None
+
+
+@dataclass
+class OutputDataset:
+    """
+    Defines an output dataset message.
+    """
+    slice_id: int = None
+    num_beams: int = None
+    num_ranges: int = None
+    num_lags: int = None
+    main_acf_shm: str = None
+    intf_acf_shm: str = None
+    xcf_shm: str = None
+
+
+@dataclass
+class ProcessedSequenceMessage:
+    """
+    Defines a message containing metadata about a processed sequence of data.
+    This message format is for communication from rx_signal_processing to data_write.
+    """
```

Comment:
I was wrong. It was used in the C++ dsp code, not the Python code. However, it was never unpacked by data_write, so really it was an unused field. It looks like it may have been used for testing brian at one point.